docs/samples-and-tutorials/index.md

```diff
@@ -56,6 +56,10 @@ This sample demonstrates many of the problems that can be solved by using Expres
 
 This series of samples demonstrate many of the features of Language Integrated Query (LINQ). The [completed sample](https://github.com/dotnet/samples/tree/master/core/linq/csharp) is available in the dotnet/samples repository on GitHub.
 
+**Microsoft Office PIA Sample**
+
+This [sample](https://github.com/dotnet/samples/tree/master/core/extensions/ExcelDemo) demonstrates the consumption of [Microsoft Office PIAs](https://docs.microsoft.com/visualstudio/vsto/office-primary-interop-assemblies?view=vs-2017) in .NET Core.
```

Comment:
Suggestion:
 
```suggestion
The [ExcelDemo](https://github.com/dotnet/samples/tree/master/core/extensions/ExcelDemo) sample demonstrates the consumption of [Microsoft Office PIAs](/visualstudio/vsto/office-primary-interop-assemblies) in .NET Core.
```

Reply:
Also, should we state this is using .NET Core 3?

Reply:
I am not against that. My desire was to keep those details in the project, but if it makes more sense to put that here sure. Opinion?